mne_bids/write.py

```diff
@@ -1244,11 +1244,23 @@ def make_dataset_description(*, path, name, hed_version=None,
 
 
 @verbose
-def write_raw_bids(raw, bids_path, events_data=None, event_id=None,
-                   anonymize=None, format='auto', symlink=False,
-                   empty_room=None, allow_preload=False,
-                   montage=None, acpc_aligned=False,
-                   overwrite=False, verbose=None):
+def write_raw_bids(
+    raw,
+    bids_path,
+    *,
```

Comment:
This is not backward compatible. I would add events_data as the end of the parameter list like done in the good old days

Reply:
Addressed in [65f181d](https://github.com/mne-tools/mne-bids/pull/1054/commits/65f181d949c23d75d01d8a8a1859b0ba94947f57), better now..? I'm still introducing kw-only arguments, but now only after the 4th parameter